packages/block-library/src/embed/test/index.native.js

```diff
@@ -816,4 +816,36 @@ describe( 'Embed block', () => {
 
 		expect( cannotEmbedText ).toBeDefined();
 	} );
+
+	it( 'insert embed from slash inserter', async () => {
+		const embedBlockSlashInserter = '/Embed';
+		const { getByPlaceholderText, getByA11yLabel } = await initializeEditor(
+			{
+				initialHtml:
+					'<!-- wp:paragraph --><p></p><!-- /wp:paragraph -->',
+			}
+		);
+
+		const paragraphText = getByPlaceholderText( 'Start writing…' );
+		fireEvent( paragraphText, 'focus' );
+		fireEvent( paragraphText, 'onChange', {
+			nativeEvent: {
+				eventCount: 1,
+				target: undefined,
+				text: embedBlockSlashInserter,
+			},
+		} );
+
+		fireEvent.press(
+			await waitFor( () => getByA11yLabel( 'Embed block' ) )
+		);
+
+		const block = await waitFor( () =>
```

Comment:
> I am wondering if a prop is missing from the mocked RCTAztecView that needs to be implemented, so that it can be rendered.

Interesting, I'll investigate this further in case we need to add extra functionality in the mock.

Reply:
After digging into how the autocomplete works, I found out that we had to trigger a different event in the `RichText` component as well as set the paragraph block as the default block, which is required by the autocomplete logic (I'll add more details about this topic in the file changes).